src/content/special-interest-groups/neuroimaging/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Introduction
+subtitle: 'Community of researchers using Nextflow and nf-core for Neuroimaging'
+groupName: Neuroimaging
+leads:
+  - AlexVCaron: Alex Valcourt Caron
+  - arnaudbore: Arnaud Boré
+
+pipelines:
+  - tractoflow
```

Comment:
this is not yet an nf-core pipeline is it? in that case this will generate a broken link in the sidebar I am afraid

Reply:
True, it's not yet, I'll remove